src/terms/xc.jl

```diff
@@ -323,6 +312,7 @@ function LibxcDensity(basis, max_derivative::Integer, ρ::RealFourierArray, ρsp
     LibxcDensity(basis, max_derivative, ρ_real, ∇ρ_real, σ_real)
 end
 
+# See libxc documentation
```

Comment:
Not really, rather see Libxc.jl documentation (which basically does not yet exist ;) ... I should write it at some point)